Yoast/yoast-components
# Removed helpTextButton from SnippetPreview

## Summary

This PR can be summarized in the following changelog entry:

* Removed the helpTextButton from the SnippetPreview as it has been replaced by a description that is always present. 

## Test instructions

This PR can be tested by following these steps:

* Check out this branch, see that the helpTextButton now (really) is gone. 

Fixes #698
